fundingmanager_test.go

```diff
@@ -304,8 +306,8 @@ func createTestFundingManager(t *testing.T, privKey *btcec.PrivateKey,
 		ReportShortChanID: func(wire.OutPoint, lnwire.ShortChannelID) error {
 			return nil
 		},
-		ZombieSweeperInterval: 1 * time.Minute,
-		ReservationTimeout:    10 * time.Minute,
+		ZombieSweeperInterval: 1 * time.Hour,
+		ReservationTimeout:    100 * time.Millisecond,
```

Comment:
since you are controlling when the zombie sweeper run, you can make this as small as possible. This let us make the needed sleeps even tighter.

Reply:
Done